### DISCUSSION TITLE
RuleType Spike: Number of events of current and reference timeframe in alerta alert.

### DISCUSSION THREAD

**Discussion by ulmako** [jertel/elastalert2#1541] [Q&A] [answered]
Hi,

I'm using the rule type Spike and send the alerts to _alerta_.

Since the an alert for the Spike type does not contain one single event, I want to send a summary in the alert containing information about the current timeframe and the reference timeframe (i.e. number of events in both timestamps)

If I understand the code correctly [(code)](https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/alerta.py#L83) , I can only use information from the first match with Python-like string syntax `{<field>}` in the fields.

Is it possible to also get information from the alert?

Thanks and regards
Ulrich

**Comment by jertel** [comment, marked as answer]
Have you tried using `spike_count` and `reference_count` from the match?

**Reply by ulmako** [reply]
This works! Thanks a lot.

I couldn't find the two values in the documentation. Is there a list with all the fields of a match?

**Reply by jertel** [reply]
I had to look at the source code to find those, and those are the only ones specific to spike rules. Feel free to submit a PR with a doc update to mention those. Thanks!

**Reply by ulmako** [reply]
https://github.com/jertel/elastalert2/pull/1542

**Reply by jertel** [reply]
Thank you!
